pkg/infra/tracing/tracing.go

```diff
@@ -381,6 +381,15 @@ func TraceIDFromContext(ctx context.Context, requireSampled bool) string {
 	return spanCtx.TraceID().String()
 }
 
+func ServerTimingForSpan(span trace.Span) string {
```

Comment:
NIT – the naming of this function is a bit confusing (maybe because I'm not super well-versed in OTel terminology) – it implies it has something to do with _timing_, whereas in reality is just constructs the `traceparent` value from the given span.

I don't think it's a huge issue so feel free to leave this as-is.

Reply:
leaving as is, thank you